velox/common/memory/MappedMemory.h

```diff
@@ -409,14 +409,10 @@ class MappedMemory : public std::enable_shared_from_this<MappedMemory> {
   // limit of 'this'. This function is not virtual but calls the virtual
   // functions allocate and allocateContiguous, which can track sizes and
   // enforce caps etc.
-  virtual void* FOLLY_NULLABLE
-  allocateBytes(uint64_t bytes, uint64_t maxMallocSize = kMaxMallocBytes);
+  virtual void* FOLLY_NULLABLE allocateBytes(uint64_t bytes);
```

Comment:
Would you update method comments? They still refer to maxMallocSize.

Reply:
Updated. Thanks!